docs/features/firefox-accounts/ecosystem-telemetry.md

```diff
@@ -0,0 +1,17 @@
+---
+id: ecosystem-telemetry
+title: Account Ecosystem Telemetry
+sidebar_label: Ecosystem Telemetry
```

Comment:
The full three-word name "Account Ecosystem Telemetry" is too long to fit in the sidebase, and context makes it obvious that this is related to accounts.